public/doc/swagger-2.yaml

```diff
@@ -581,13 +581,13 @@ parameters:
     in: path
     description: The Portfolio ID
     required: true
-    type: integer
+    type: string
   PortfolioItemID:
     name: portfolio_item_id
     in: path
     description: The Portfolio Item ID
     required: true
-    type: integer
+    type: string
```

Comment:
@syncrou Do you also need to make this change in the app/models for these objects so that the JSON payload coming back has the value as string.

Reply:
@syncrou Maybe we need to get this in the same way that the topological service does it.
In a separate PR

Reply:
@mkanoor - Ok, I'll get Brandon's code in on another PR.